cpp/docs/DEVELOPER_GUIDE.md

```diff
@@ -373,7 +373,7 @@ namespace detail{
     void external_function(..., rmm::cuda_stream_view stream){
         // Implementation uses the stream with async APIs.
         rmm::device_buffer buff(...,stream);
-        CUDA_TRY(cudaMemcpyAsync(...,stream.value()));
+       CUDF_CUDA_TRY(cudaMemcpyAsync(...,stream.value()));
```

Comment:
Good job finding the doc'd ones.
```suggestion
        CUDF_CUDA_TRY(cudaMemcpyAsync(...,stream.value()));
```
Looks like the spacing is off by one on this line.

Reply:
Good catch. I had a mistake in my find-replace regex at first, which removed one character to the left. I thought I caught/fixed all of those but missed this one.